src/sankey.js

```diff
@@ -51,12 +65,13 @@ function computeLinkBreadths({nodes}) {
   }
 }
 
-export default function Sankey() {
+function Sankey(orient) {
```

Comment:
The word `orient` evokes "The Orient" (as opposed to "The Occident"). Perhaps `orientation` would be better?
```suggestion
function Sankey(orientation) {
```